### PR TITLE
Feature/square 1 home ui

### PR DESCRIPTION
**Pull request by TimaVysotski**

### CODE REVIEW

File: android/app/src/main/java/com/square/MainApplication.java [hunk 1]
```diff
@@ -4,6 +4,8 @@
 import android.content.Context;
 import com.facebook.react.PackageList;
 import com.facebook.react.ReactApplication;
+import com.cmcewen.blurview.BlurViewPackage;
```
Comment (MeleshkevichAleksey) [inline android/app/src/main/java/com/square/MainApplication.java]: I think you should import libs according alphabet.

File: src/container/components/button/Switch.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import Label from '../text/Label';
+import sizes from '../../../theme/sizes';
+
+interface SwitchProps {
+  children?: Array<ReactElement> | ReactElement;
+  fontSize?: number;
+  color?: string;
+  flex?: number | boolean;
+  text?: string;
+  mt?: number | boolean;
```
Comment (MeleshkevichAleksey) [inline src/container/components/button/Switch.tsx]: Interface might contain intuitive understandable fields, or they should be commented.

File: src/container/components/button/Switch.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  mb?: number | boolean;
+  mr?: number | boolean;
+  ml?: number | boolean;
+}
+
+const StyledSwitchView = styled(View)`
+  flex-direction: row;
+  flex: ${({ flex = 1 }: SwitchProps): number => Number(flex)};
+  ${({ color }: SwitchProps): string => `backgroundColor: ${color}`}
+  margin-top: ${({ mt = 0 }: SwitchProps): number => Number(mt) * sizes.baseMargin};
```
Comment (MeleshkevichAleksey) [inline src/container/components/button/Switch.tsx]: Maybe component location settings should be separated from component declaration?

File: src/theme/colors.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
+interface SquareColorsProps {
+  GRADIENT: Array<string>;
```
Comment (MeleshkevichAleksey) [inline src/theme/colors.ts:2]: Why not camelcase like you write it before?

File: src/container/components/button/Switch.tsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+import sizes from '../../../theme/sizes';
+
+interface SwitchProps {
+  children?: Array<ReactElement> | ReactElement;
+  fontSize?: number;
+  color?: string;
+  flex?: number | boolean;
+  text?: string;
+  mt?: number | boolean;
+  mb?: number | boolean;
```
Comment (MeleshkevichAleksey) [inline src/container/components/button/Switch.tsx]: Interface might contain intuitive understandable fields, or they should be commented.

File: android/app/src/main/java/com/square/MainApplication.java (hunk 1 above)
Comment (TimaVysotski) [reply android/app/src/main/java/com/square/MainApplication.java]: Sry, but this system file and i doesn't have access to style of importing.

File: src/container/components/button/Switch.tsx (hunk 3 above)
Comment (TimaVysotski) [reply src/container/components/button/Switch.tsx]: This is initialization of component style.

File: src/theme/colors.ts (hunk 4 above)
Comment (TimaVysotski) [reply src/theme/colors.ts:2]: True.

but this is Theme constants.

File: src/container/components/button/Switch.tsx (hunk 5 above)
Comment (TimaVysotski) [reply src/container/components/button/Switch.tsx]: This is intuitive understandable name of variable.
This style use Material UI.
U can check it https://material-ui.com/
mt = margin-top

**Pull request merged**